src/mnd/commands/open.cr

```diff
@@ -0,0 +1,37 @@
+module Mnd
+  class Commands::Open < Commands::Base
+    summary "Conveniently opens services based on repository context"
+    usage <<-EOF
+	    mnd open buildkite # opens repo/branch in buildkite
+	    mnd open github # opens repo in github
+	    mnd open rollbar # opens repo in rollbar
+	  EOF
+
+    def perform
+      # ie. "https://github.com/mynewsdesk/mynewsdesk.git\n" -> "mynewsdesk"
+      current_repo = File.basename(`git remote get-url origin`).chomp(".git\n")
+
+      if current_repo.empty?
```

Comment:
There are other errors you can get here, like if you don't have an origin configured. But perhaps that is not necessary to handle..

Reply:
`current_repo` will be empty string whatever the output to STDERR since backticks only capture STDOUT. So it's kind of a catch all. I tried for formulate the error message in a way that would hint to the developer that looking at the origin was involved.